site/phpstan-baseline.neon

```diff
@@ -33,7 +33,7 @@ parameters:
 
 		-
 			message: "#^Call to function is_null\\(\\) with app\\\\models\\\\User will always evaluate to false\\.$#"
-			count: 3
+			count: 4
```

Comment:
We shouldn't be adding more ignored errors unless strictly necessary.  The new error should be fixed rather than ignored.  I suspect the issue is coming from line 162 of [DockerInterfaceController.php](https://github.com/Submitty/Submitty/pull/8845/files#diff-bfd0eb036127c1252522af67074ab2cfa90757bd093a6bf5947cdca7235e6060).

Reply:
Reviewing this it seems that there is no need of is_null check for the removeImage function since it is being already done by the showDockerInterface function.